txinput.go

```diff
@@ -34,6 +45,28 @@ func NewInputFromBytes(bytes []byte) (*Input, int, error) {
 	}, totalLength, nil
 }
 
+// NewInputFrom builds and returns a new input from the specified UTXO fields, using the default
+// finalised sequence number (0xFFFFFFFF). If you want a different nSeq, change it manually
+// afterwards.
+func NewInputFrom(prevTxID string, vout uint32, prevTxLockingScript string, satoshis uint64) (*Input, error) {
```

Comment:
shouldn't these functions go into the [inputs.go file](https://github.com/libsv/go-bt/blob/master/input.go) instead of here? i think what we were going for was to try an minimise the number of ways users could do things. so in v1 the user had to create the input himself and then add that input to the tx, whereas with v2 the point is to use the .From() method to create the input as part of the tx straight away since having an input alone isn't much use is it? maybe we can keep this func but just put it in inputs.go and make in private? what do you guys think? @theflyingcodr @Tigh-Gherr ?